php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S4721.json

```diff
@@ -0,0 +1,21 @@
+{
+  "title": "Executing OS commands is security-sensitive",
+  "type": "CODE_SMELL",
```

Comment:
I think the type should be `SECURITY_HOTSPOT`

Reply:
Changed in 25f788d